.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,41 @@
+name: PR Guideline Checks
+
+on:
+  pull_request:
+    types: [opened, reopened, edited]
```

Comment:
Workflow implications:

* We get a @github-actions comment every time the PR body/title is edited. Including every time a Checklist task is ticked off.
* Pushing new commits doesn't trigger the PR check action. This means (I think?) that a subsequent push can mask a failing PR check in the UI.

Reply:
I think the action-run-for-each-update is fine.

In addition, pushing now _does_ trigger the action because that's how we can pick up pushes for `changes/*.md`.